symfony/src/Service/EmailNotificationService.php

```diff
@@ -27,6 +27,12 @@ class EmailNotificationService
     protected $filterSummaries = false;
     protected $logEntity;
 
+    abstract protected function getSearchParams($id, $lastDeactivate);
+
+    protected function filterSummaries($summaries) {
```

Comment:
This appears to be unreachable code anyway, but there wasn't a method defined for what should happen. Correct response may be to remove the logic entirely.